pkg/drivers/kic/oci/network.go

```diff
@@ -70,8 +81,25 @@ func dockerGatewayIP(profile string) (net.IP, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "get network bridge")
 	}
+	networksOutput := strings.TrimSpace(rr.Stdout.String())
+	networksSlice := strings.Fields(networksOutput)
+	// Look for the minikube container within each docker network
+	for _, net := range networksSlice {
+		// get all containers in the network
+		rs, err := runCmd(exec.Command(Docker, "network", "inspect", net, "-f", "'{{range $k, $v := .Containers}}{{$v.Name}} {{end}}'"))
```

Comment:
Remove the inner single-quotes from the template. Otherwise, on UNIX, it adds single apostrophes to the first and last network name.